tornado-runtime/src/main/java/uk/ac/manchester/tornado/runtime/common/TornadoOptions.java

```diff
@@ -43,7 +43,7 @@ public class TornadoOptions {
      * Priority of the OpenCL Backend. The higher the number, the more priority over
      * the rest of the backends.
      */
-    public static final int OPENCL_BACKEND_PRIORITY = Integer.parseInt(Tornado.getProperty("tornado.opencl.priority", "10"));
+    public static final int OPENCL_BACKEND_PRIORITY = Integer.parseInt(Tornado.getProperty("tornado.opencl.priority", "12"));
```

Comment:
Why is this value changed?

Reply:
@jjfumero did this change

Reply:
This was for testing. @mikepapadim , please revert to the previous values